src/modules/cmdpal/extensionsdk/nuget/Microsoft.CmdPal.Extensions.SDK.nuspec

```diff
@@ -0,0 +1,35 @@
+<?xml version="1.0"?>
+<package xmlns="http://schemas.microsoft.com/packaging/2013/05/nuspec.xsd">
+  <metadata minClientVersion="2.5">
+    <id>Microsoft.CmdPal.Extensions.SDK</id>
+    <version>1.0.0</version>
```

Comment:
should this be 0.0.1?

Reply:
This gets set by the -pack command in BuildSDKHelper.ps1.  But I did change this to 0.0.0.  At least this way if something goes wrong with the version number, it will be immediately evident :)